src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/auth/AuthConstants.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.api.rest.auth
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.core.{Authentication, GrantedAuthority}
+import org.springframework.stereotype.Component
+import za.co.absa.hyperdrive.trigger.configuration.application.AuthConfig
+
+@Component("authConstants")
+class AuthConstants @Autowired()(authConfig: AuthConfig) {
```

Comment:
Hmm, it's a bit a strange name `Constants` for a class with methods. Maybe `AuthUtils`?

Reply:
Yeah true, I've copied it and it originally contained constants.
Done.